### PR TITLE
Added help , Fixed #10

### LINKED ISSUE #10: No help
The "Help" toolbar option opens a page that says "under construction".

Also, it points to DocumentViewer's old home, the URL needs to be updated.

### PR DESCRIPTION
**Pull request by vanishakesswani**
* Added a Help PDF
* Updated link to help page .

### CODE REVIEW

**Comment by humdingerb** [comment]
Opening the browser to go online and downloading the PDF that then will presumably be opened with DocumentViewer is unnecessarily complicated and limiting. Why not just open the PDF in the app?

The PDF's English could be improved... I can help you with that.

You can get a nicer big icon by exporting in Icon-O-Matic as .SVG, then open it in WonderBrush, resize it and export as .PNG.

Finally, you should add the editable source for the PDF. Doesn't matter if the editing has to be done in another OS because Haiku doesn't have the necessary apps. Use an open format, if possible.

**Comment by humdingerb** [comment]
For the help PDF and ODF, I'd say create a folder "documentation" on the top level beside "application".

To open in the app itself, you can call MainWindow::_OpenFile(), see https://github.com/HaikuArchives/DocumentViewer/blob/master/application/MainWindow.cpp#L250

To get the right path, have a look at https://github.com/HaikuArchives/BurnItNow/blob/master/Source/BurnWindow.cpp#L402
You have to use FindPaths() because the app can be installed in /system and ~/config and the documentation will end up in either hierarchy when installed.

**Comment by vanishakesswani** [comment]
These code is rocket science for me . I haven't ever learned coding in school.
For this you will have to help me out with step by step guidance .

**Comment by vanishakesswani** [comment]
Ohh! Will keep that in mind always :) . Thanks !

**Comment by pulkomandy** [comment]
Let's see about opening a file.

Humdinger pointed you to the OpenFile method. You can see it takes parameters: a path to the file, a file type, a password, and a page number.
The last two are easy: we want no password and open on the first page.

The type of the file is "pdf", because, it is a PDF file.

So, let's call the method with these arguments:

    _OpenFile("help.pdf", "pdf", "", 0);

Now, this will look for the file named "help.pdf". The problem is, we didn't tell the app *where* to look for the file. This is what FindPaths can help us to do: it will try several locations and pick the most approrpiate one. I see that the example in BurnItNow is not the simplest way to do it, however, so let me write some example...

**Comment by pulkomandy** [comment]
Ok, so let's see how to use BPathFinder.
First we need to create it:

    BPathFinder pathFinder;

We now have an object called pathFinder, and we can ask it to do things. For example, ask it to look for a file in the documentation directory. But first, we need a place to store the result:

    BPath path;

Ok, now let's do the thing:

    pathFinder.FindPath(B_FIND_PATH_DOCUMENTATION_DIRECTORY,
		"packages/documentviewer/help.pdf", path, B_FIND_PATH_EXISTING_ONLY);

This will fill in our *path* with the complete path to the file. The last argument makes sure the file exists, otherwise the pathFinder will say there is an error.

And finally we can give that path to _OpenFile (we need to convert it to a string using Path() because that's what _OpenFile expects):

    _OpenFile(path.Path(), "pdf", "", 0);

I hope everything is clear now.

**Comment by vanishakesswani** [comment]
Why have we entered the argument for page number as 0 even though we want the initial page number to be 1 ?

**Comment by vanishakesswani** [comment]
You just made programming look quite easy  .

File: application/MainWindow.cpp [hunk 1]
```diff
@@ -156,8 +156,13 @@ MainWindow::MessageReceived(BMessage* message)
 
         case MSG_HELP:
         {
-        	char const* args[] = {"http://haikuarchives.github.io/DocumentViewer/help.html", 0};
-        	be_roster->Launch("text/html", 1, args);
+        	BPathFinder pathFinder;
+			BPath path;
```
Comment (pulkomandy) [inline application/MainWindow.cpp]: What happened to the indentation here? These lines should start on the same column.

**Comment by vanishakesswani** [comment]
The indentation looks good in the compiler , don't why it's not the same on github .
Used GitHub editor and fixed indentation . Even on github editor there is no extra line between 

> BPathFinder pathFinder;
BPath path;

but the final final still has a extra line between them .1

**Comment by pulkomandy** [comment]
Ok, we are good to go on the code side, I will let humdinger and others decide on the document contents before we merge this.

**Comment by humdingerb** [comment]
: Opens a friendly pop-up which helps to browse your file.
-> Opens a file dialog to load a document.

: Show’s you the thumbnail images.
-> Shows the thumbnails of the document.

: Search anything within the file using this tool.
-> Find text in the document.

: Give’s you access to feature’s like changing page, printing, opening a new file and closing current one.
-> The home panel gives access to features like jumping to a page, printing, opening a new document and the current one.

: Give’s a print command .
-> Prints the current document.

: Give’s you access to tools like Zoom In, Zoom Out, Fit to width, Fit to height, full screen window etc .
-> The view panel gives access to tools like Zoom in/out, Fit to page width, Fit to page height, Fullscreen mode etc.

: Zoom Out
-> Zoom out

: Zoom In
-> Zoom in

: Fit to width
-> Fit to page width

: Fit to height
-> Fit to page height

: Takes you to this guide in your browser.
-> Opens this help

: Takes you to the support page in your browser.
-> Opens the project website 


Rest of the application is self-explanatory in case of any help ping us @
irc.freenode.org Haiku .

I'll just remove the last paragraph.

**Pull request merged**